package.json

```diff
@@ -8,6 +8,8 @@
     "@babel/core": "^7.6.2",
     "@storybook/html": "^5.2.1",
     "babel-loader": "^8.0.6",
+    "twig": "^1.13.3",
```

Comment:
Not sure how an export would work. Could be that the whole thing is added to NPM and this is should be under `dependencies`?

Reply:
Depends whether `dependencies`  / `devDependencies` refers to the built storybook (in which case there are no `dependencies`) or the npm package containing the patterns (in which case it would go in `dependencies`).